Eor bubble fixes

aelanman opened the pull request:
The most significant change in this branch is that I _correctly_ form the uvf cube in eor_bubble_sim so that the degridding step actually loops over frequency. For the sake of computational speed, I also switch back to using visibility_degrid instead of its wrapper (vis_source_model) in vis_simulate. There was some discussion back in January about the differences of these two, and it might be worth reexamining vis_simulate.

isullivan commented on simulation/eor_bubble_sim.pro:
```diff
@@ -55,27 +55,24 @@ H5S_CLOSE, dspace_id_eor
    model_tmp=Ptrarr(n_pol,/allocate)
    *model_tmp[0] = *model_stokes_arr[fi]
-   FOR pol_i=1,n_pol-1 DO *model_tmp[pol_i]=Fltarr(obs.dimension,obs.elements)
+   *model_tmp[1] = fltarr(obs.dimension, obs.elements)
```
This will cause an error if `n_pol` is anything other than 2. I'd suggest reverting this one line

isullivan commented on simulation/vis_simulate.pro:
```diff
@@ -263,18 +263,18 @@ FUNCTION vis_simulate,obs,status_str,psf,params,jones,skymodel,file_path_fhd=fil
           print, 'model loop num, timing(s):'+ number_formatter(fi) + ' , ' + number_formatter(model_timing)
           
-          for pol_i=0,n_pol-1 do (*vis_model_arr[pol_i])[fi,*] = (*this_model_ptr[pol_i])[fi,*]
+;          for pol_i=0,n_pol-1 do (*vis_model_arr[pol_i])[fi,*] = (*this_model_ptr[pol_i])[fi,*]
```
If you are leaving commented-out code to make it easier to switch back later, please make sure it is all in one block and clearly marked. Also, please indent your comments to the same level as the surrounding code (but don't indent the commented-out code).

aelanman commented:
I've addressed your comments @isullivan . I switched back to the pol_i loop in eor_bubble_sim so that all pols are initialized to fltarr(dimension, elements), in case n_pol != 2. I removed the commented code.

I would still like to find a time to update and improve vis_simulate.
